Solutions/BitSight/Parsers/BitSightCompanyDetails.txt

```diff
@@ -0,0 +1,114 @@
+                InSpmPortfolio = column_ifexists('in_spm_portfolio_b', ''),
+                Industry = column_ifexists('industry_s', ''),
+                IndustrySlug = column_ifexists('industry_slug_s', ''),
+                Ipv4Count = column_ifexists('ipv4_count_d', ''),
+                IsBundle = column_ifexists('is_bundle_b', ''),
+                IsCsp = column_ifexists('is_csp_b', ''),
+                IsMycompMysubsBundle = column_ifexists('is_mycomp_mysubs_bundle_b', ''),
+                IsPrimary = column_ifexists('is_primary_b', ''),
+                IsUnsampledAllowed = column_ifexists('is_unsampled_allowed_b', ''),
+                Name = column_ifexists('name_s', ''),
+                PeopleCount = column_ifexists('people_count_d', ''),
+                PermissionCanAnnotate = column_ifexists('permissions_can_annotate_b', ''),
+                PermissionCanDownloadCompanyReport = column_ifexists('permissions_can_download_company_report_b', ''),
+                PermissionCanEnableVendorAccess = column_ifexists('permissions_can_enable_vendor_access_b', ''),
+                PermissionCanViewCompanyReports = column_ifexists('permissions_can_view_company_reports_b', ''),
+                PermissionCanViewForensics = column_ifexists('permissions_can_view_forensics_b', ''),
+                PermissionCanViewInfrastructure = column_ifexists('permissions_can_view_infrastructure_b', ''),
+                PermissionCanViewIpAttributions = column_ifexists('permissions_can_view_ip_attributions_b', ''),
+                PermissionCanViewServiceProviders = column_ifexists('permissions_can_view_service_providers_b', ''),
+                PermissionsHasControl = column_ifexists('permissions_has_control_b', ''),
+                PrimaryDomain = column_ifexists('primary_domain_s', ''),
+                RatingIndustryMedian = column_ifexists('rating_industry_median_s', ''),
+                Ratings = column_ifexists('ratings_s', ''),
+                RelatedCompanies = column_ifexists('related_companies_s', ''),
+                SearchCount = column_ifexists('search_count_d', ''),
+                ServiceProvider = column_ifexists('service_provider_b', ''),
+                Shortname = column_ifexists('shortname_s', ''),
+                Sparkline = column_ifexists('sparkline_s', ''),
+                SubIndustry = column_ifexists('sub_industry_s', ''),
+                SubIndustrySlug = column_ifexists('sub_industry_slug_s', ''),
+                SubscriptionType = column_ifexists('subscription_type_s', ''),
+                SubscriptionTypeKey = column_ifexists('subscription_type_key_s', ''),
+                ComplianceClaimCertifications = column_ifexists('compliance_claim_certifications_s', ''),
+                ComplianceClaimTrustPage = column_ifexists('compliance_claim_trust_page_s', ''),
+                type = column_ifexists('type_s', ''),
+                Type = column_ifexists('Type', '')
```

Comment:
Many fields are not present in the sample data and thus cannot be tested.

Reply:
@devikamehra I have updated correct sample data.

Reply:
Thank you for updating the sample data.
Type is a reserved field and thus will always be there in the custom field. We do not need column_ifexists.